test/k8sT/demos.go

```diff
@@ -64,6 +64,8 @@ var _ = Describe("K8sDemosTest", func() {
 
 		ExpectCiliumReady(kubectl)
 		ExpectKubeDNSReady(kubectl)
+		By("Waiting for all etcd-operator pods are ready")
+		ExpectETCDOperatorReady(kubectl)
```

Comment:
Shouldn't the etcd operator be ready before cilium is ready? How cilium can be ready without etcd be ready?

Reply:
Both depend on each other, it doesn't matter which one is ready first, what matters is both are ready.